Makefile.mjs

```diff
@@ -0,0 +1,60 @@
+// eslint-disable-next-line import/no-extraneous-dependencies
+import {$} from 'execa';
+
+const commands = new Set(process.argv);
+
+if (commands.has('init')) {
```

Comment:
Does this file structure mean that `node Makefile.mjs create-secrets create-queue start create-bucket init 
 upload-file  create-parameters ` is a legal command? Is this the intent?

I can see that all commands if blocks have an `await` so things won't trip up, but should they just return/break after each one if chaining is not intended here?

Reply:
As per discussion: That command style is legal, however it hasn't been documented and isn't supported.

*I did find it useful during development